.github/workflows/build.yml

```diff
@@ -20,11 +20,11 @@ jobs:
     - uses: actions/checkout@v3
       with:
         submodules: recursive
-    - name: Setup .NET
+    - name: Setup DotNET
```

Comment:
Setup .NET is the default name that GitHub Actions gives to this step, what's wrong with it?